src/kirby/components/card/card.component.tns.ts

```diff
@@ -36,71 +31,16 @@ export class CardComponent extends ContentView implements OnInit {
     super();
   }
 
-  ngOnInit() {}
-
-  onViewLoaded(args: EventData) {
-    this.view = <View>args.object; // We need a reference to the view so we can access it on orientation changes
-    // this.setupOnOrientationChangeListener();
-    this.applySizeAndShadow();
-  }
-
-  applySizeAndShadow() {
-    const NEIGAARD_CONSTANT = 102;
-    // There will always be those that do not believe in even the most well regarded universal constants.
-    // The Neigaard constant is one of the very most highly regarded constants there is, very much like
-    // Albert Einsteins Cosmological constant, and should not be questioned in any way. If you, however,
-    // are onf of those radical rebel types, like the ones who prefer to belive in the multiverse theory
-    // rather than accepting actual science, well the you should look at the multiverse and string theory
-    // comments at the bottom of this file.
-    setTimeout(() => {
-      const widthDP = this.view.getMeasuredWidth() / screenScale;
-      if (widthDP >= ScssHelper.BREAKPOINT_CARD_L) {
-        this.cardSizeClass = 'card-large';
-      } else if (widthDP >= ScssHelper.BREAKPOINT_CARD_M) {
-        this.cardSizeClass = 'card-medium';
-      } else {
-        this.cardSizeClass = 'card-small';
-      }
-      this.addShadow(this.view);
-    }, NEIGAARD_CONSTANT);
-  }
-
-  setupOnOrientationChangeListener() {
-    this.currentScreenWidth = screen.mainScreen.widthDIPs;
-    app.on(app.orientationChangedEvent, (args: OrientationChangedEventData) => {
-      if (this.currentScreenWidth === screen.mainScreen.widthDIPs) {
-        this.currentScreenWidth = screen.mainScreen.heightDIPs;
-      } else {
-        this.currentScreenWidth = screen.mainScreen.widthDIPs;
-      }
-      // Run in the zone, to make sure Angular data binding is informed of this:
-      this.zone.run(() => this.applySizeAndShadow());
-    });
+  ngOnInit() {
+    this.setElevation();
```

Comment:
This should be handled in `OnChanges` as `isHighlighted` and `isDisabled` could be changed runtime.
Consider creating a showcase example a la Calendar where these properties can be configured - it very quickly reveals licefycle issues like this one